### PR TITLE
[WIP][AIRFLOW-1823] added possibility to send execution date with millisec…

### PR DESCRIPTION
**Pull request by xnuinside**
…onds to task_instance_info REST API endpoint

Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-1823\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-1823
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

added compatibility with UI triggered DAGs
checked: 

screen shot 2018-11-25 at 23 19 59

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

**Comment by xnuinside** [comment]
@Fokko, @ashb, @feng-tao, @bolkedebruin, somebody please review (I don't know what's with TravisCI - strange fails). As I understand the case that we have different timestamps then trigger DAG from UI and then trigger from the cli are the some kind of incorrect behavior and 'bug' but something tells me what it's not a quick issue and fix with moving to one format (with or without milliseconds will cause some backward incapability changes) or I'm wrong? I prepare a fix that makes rest api works with two execution date formats - with milliseconds and without (as it was)

File: airflow/www_rbac/api/experimental/endpoints.py [hunk 1]
```diff
@@ -168,18 +168,21 @@ def task_instance_info(dag_id, execution_date, task_id):
...
     The format for the exec_date is expected to be
-    "YYYY-mm-DDTHH:MM:SS", for example: "2016-11-16T11:34:15". This will
+    "YYYY-mm-DDTHH:MM:SS", for example: "2016-11-16T11:34:15" or
+    "YYYY-mm-DDTHH:MM:SS.f", for example: "2016-11-16T11:34:15.680399". This will
     of course need to have been encoded for URL in the request.
     """
 
     # Convert string datetime into actual datetime
     try:
-        execution_date = timezone.parse(execution_date)
```
Comment (bolkedebruin) [inline airflow/www_rbac/api/experimental/endpoints.py:177]: You are removing timezone information here and rely on your own parsing mechanism. I don't think that's a good idea?

**Review by bolkedebruin — COMMENTED** [review]
I'm not sure about this change. Timezone information is discarded and you are implementing your own parsing mechanism while not implementing tests. What is the added value of having execution dates triggered with subsecond accuracy?

File: airflow/www_rbac/api/experimental/endpoints.py (hunk 1 above)
Comment (xnuinside) [reply airflow/www_rbac/api/experimental/endpoints.py:177]: @bolkedebruin, oh shame! I don't pay attention to what it was changed from 1.9 and just apply my fix from 1.9, sorry for that (it was - https://github.com/apache/incubator-airflow/blob/v1-9-stable/airflow/www/api/experimental/endpoints.py#L126)

so, maybe this task not actually already in 1.10? 
https://issues.apache.org/jira/projects/AIRFLOW/issues/AIRFLOW-1823

Comment (xnuinside) [reply airflow/www_rbac/api/experimental/endpoints.py:177]: yes, it works correct in 1.10, sorry for PR, I think need to close task - https://issues.apache.org/jira/projects/AIRFLOW/issues/AIRFLOW-1823

**Pull request closed without merging**